datajoint/blob.py

```diff
@@ -487,17 +487,24 @@ def pack_cell_array(self, array):
     def read_datetime(self):
         """deserialize datetime.date, .time, or .datetime"""
         date, time = self.read_value("int32"), self.read_value("int64")
+        date_str = str(date)
+        time_str = str(time)
+
         date = (
-            datetime.date(year=date // 10000, month=(date // 100) % 100, day=date % 100)
```

Comment:
Why the change? Here are some timing tests:
![image](https://user-images.githubusercontent.com/1096169/167878132-76ed9049-a3c8-4eb2-81f7-f3bed088c699.png)